website/addons/base/__init__.py

```diff
@@ -69,7 +69,7 @@ def __init__(self, short_name, full_name, owners, categories,
                  node_settings_model=None, user_settings_model=None, include_js=None, include_css=None,
                  widget_help=None, views=None, configs=None, models=None,
                  has_hgrid_files=False, get_hgrid_data=None, max_file_size=None, high_max_file_size=None,
-                 accept_extensions=True,
+                 accept_extensions=True, description='', url=None,
```

Comment:
Like I said, not a blocker for this PR in any way- more a general impression. The very existence of a constructor implies that this data lives somewhere else already (addon settings presumably)...